rialto_airflow/dags/google.py

```diff
@@ -0,0 +1,103 @@
+# An example DAG showing how to interact with Google Sheets and Google Drive using Airflow.
+# Pull these examples into your own DAGs as needed
+
+import datetime
+
+from airflow.decorators import dag, task
+from airflow.models import Variable
+from airflow.providers.google.suite.transfers.local_to_drive import (
+    LocalFilesystemToGoogleDriveOperator,
+)
+from airflow.providers.google.suite.hooks.sheets import GSheetsHook
+from airflow.providers.google.suite.hooks.drive import GoogleDriveHook
+
+gcp_conn_id = Variable.get("google_connection")
+google_drive_id = Variable.get("google_drive_id")
+google_sheet_id = Variable.get(
+    "orcid_stats_sheet_id", default_var="1smq7H5wuGBzTMsO3sAGyPd-Uz8BBlECX1xGd_0pwTrY"
```

Comment:
`orcid_status_sheet_id` is the google sheet ID to clear/write to in these examples...

the actual ID could be:

-     hardcoded in the steps (least good idea)
-     referenced like this and then setup in the airflow UI in the admin -> variables
-     referenced like this and then setup via values stored in vault and set by puppet